ntbs-service/Pages/Patients/Edit.cshtml.cs

```diff
@@ -68,17 +68,7 @@ public async Task<IActionResult> OnPostAsync(int? id)
             var notification = await service.GetNotificationAsync(id);
             await service.UpdatePatientAsync(notification, Patient);
             
-            return RedirectToPage("/ClinicalTimelines/Edit", new {id = notification.NotificationId});
-        }
-
-        public ContentResult OnPostValidatePatientProperty(string key, string value)
-        {
```

Comment:
I moved all these into ValidationModel now - makes more sense to define all the individual field validations in there I think, as they're not part of the ordinary razor page validation, but only used for dynamic validation.

Reply:
I'm super sure about this - it removes some clutter from this file, but dumps it all into another without commissioning. The change makes the generic `ValidationModel` undesrtand that there are specific aspects to the notification, e.g. patient details, sites, etc., which feels odd for a base model. 
In the current system, each page also covers its page's dynamic validation needs.
Let's discuss this one further?